package.json

```diff
@@ -96,6 +96,7 @@
     "mdn-polyfills": "^5.16.0",
     "memoize-one": "^5.0.2",
     "moment": "^2.24.0",
+    "muuri": "^0.8.0",
```

Comment:
Missing `yarn.lock`

Reply:
ah I thought this was something that wasn't push..